sql/codegen.go

```diff
@@ -457,11 +457,13 @@ def insert(table_name, eval_input_dataset, feature_column_names, predictions, in
         pred_rows.append(tuple(row))
         if len(pred_rows) == insert_batch_size:
             insert_cursor = insert_values(driver, write_conn, table_name, column_names, pred_rows)
-            insert_cursor.close()
```

Comment:
How about we change it to `close(driver, insert_curser)` and in the `db.py`, we add
```
def close(driver, insert_curser):
    if driver != "maxcompute":
        insert_cursor.close()
```
So that we can avoid `if driver != "maxcompute":` in the codegen_template.